src/Compilers/CSharp/Portable/Symbols/Synthesized/Records/SynthesizedRecordEqualityContractProperty.cs

```diff
@@ -55,7 +55,7 @@ public override SyntaxList<AttributeListSyntax> AttributeDeclarationSyntaxList
         protected override Location TypeLocation
             => ContainingType.Locations[0];
 
-        protected override SourcePropertyAccessorSymbol CreateGetAccessorSymbol(bool isAutoPropertyAccessor, BindingDiagnosticBag diagnostics)
+        protected override SourcePropertyAccessorSymbol CreateGetAccessorSymbol(BindingDiagnosticBag diagnostics)
         {
             return SourcePropertyAccessorSymbol.CreateAccessorSymbol(
```

Comment:
>CreateAccessorSymbol [](http://example.com/codeflow?start=48&length=20)

It looks like this is the only call site for this method and its implementation is very specialized for this class. I think it would be good to remove it and simply inline its body here. After that we should be able to make ```GetAccessorSymbol``` private.